nodestream_plugin_k8s/commands/list_command.py

```diff
@@ -33,13 +39,25 @@ def display_as_json(self, items_to_display: Iterable[PipelineDesiredState]):
         )
         self.line(json_string)
 
+    def add_development_args(self, items_to_display):
+        for item in items_to_display:
+            item.perpetual_concurrency = (
+                item.perpetual_concurrency if item.debug_enabled else 0
+            )
+            item.suspend = False if item.debug_enabled else True
+            yield item
```

Comment:
This method, as it is today, should belong on `item` in my opinion.  It has [feature envy](https://refactoring.guru/smells/feature-envy)

Reply:
Also can you document this behavior in the README?